<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>809</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>To Do List</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
    <item>
     <layout class="QHBoxLayout" name="topHLayout" stretch="1,4,1">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QComboBox" name="jobSortCoBox">
        <property name="minimumSize">
         <size>
          <width>256</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Open</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Completed</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Everything</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="jobNameEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="jobStatCoBox">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Have not looked at</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pending</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>In Progress</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Complete</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="taskLayout">
        <item>
         <widget class="QLabel" name="taskListLabel">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Task List</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="jobListWidget"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <layout class="QVBoxLayout" name="rightLayout">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget">
           <layout class="QVBoxLayout" name="DescriptionLayout">
            <item>
             <widget class="QLabel" name="descriptionLabel">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Description</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="jobTextEdit"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="verticalLayoutWidget_2">
           <layout class="QVBoxLayout" name="journalLayout">
            <item>
             <widget class="QLabel" name="JournalLabel">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Journal</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="journalListWidget"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="journalLineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="addJournalBtn">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewTask"/>
   <addaction name="actionSaveTask"/>
   <addaction name="separator"/>
   <addaction name="actionOpenToDoList"/>
   <addaction name="actionNewToDoList"/>
  </widget>
  <action name="actionNewTask">
   <property name="text">
    <string>New Task</string>
   </property>
   <property name="toolTip">
    <string>add a new task to the to do list</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSaveTask">
   <property name="text">
    <string>Save Task</string>
   </property>
   <property name="toolTip">
    <string>Save the current task</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpenToDoList">
   <property name="text">
    <string>Open To Do List</string>
   </property>
   <property name="toolTip">
    <string>Open another to do list</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNewToDoList">
   <property name="text">
    <string>New To Do List</string>
   </property>
   <property name="toolTip">
    <string>Make a new to do list</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>jobNameEdit</tabstop>
  <tabstop>jobStatCoBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
